modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/DDMFormFieldTemplateContextContributorUtilTest.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.dynamic.data.mapping.form.field.type.internal.util;
+
+import com.liferay.dynamic.data.mapping.form.field.type.internal.checkbox.CheckboxDDMFormFieldTemplateContextContributor;
+import com.liferay.dynamic.data.mapping.model.DDMFormField;
+import com.liferay.dynamic.data.mapping.test.util.BaseDDMFormFieldTypeSettingsTestCase;
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.test.rule.LiferayUnitTestRule;
+
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+/**
+ * @author Pedro Leite
+ */
+public class DDMFormFieldTemplateContextContributorUtilTest
+	extends BaseDDMFormFieldTypeSettingsTestCase {
+
+	@ClassRule
+	@Rule
+	public static final LiferayUnitTestRule liferayUnitTestRule =
+		LiferayUnitTestRule.INSTANCE;
+
+	@Test
+	public void testGetLocalesMap() {
+		CheckboxDDMFormFieldTemplateContextContributor
+			checkboxDDMFormFieldTemplateContextContributor =
+				new CheckboxDDMFormFieldTemplateContextContributor();
+
+		DDMFormField ddmFormField = new DDMFormField("field", "checkbox");
+
+		ddmFormField.setDDMForm(getDDMForm());
+
+		Map<String, Object> parameters =
+			checkboxDDMFormFieldTemplateContextContributor.getParameters(
+				ddmFormField, createDDMFormFieldRenderingContext());
+
+		JSONArray availableLocalesJSONArray = (JSONArray)parameters.get(
+			"availableLocales");
+		JSONObject defaultLocaleJSONObject = (JSONObject)parameters.get(
+			"defaultLocale");
+		JSONObject editingLocaleJSONObject = (JSONObject)parameters.get(
+			"editingLocale");
+
+		Assert.assertEquals(
+			availableLocales.size(), availableLocalesJSONArray.length());
+		Assert.assertEquals("en_US", defaultLocaleJSONObject.get("localeId"));
+		Assert.assertEquals("en_US", editingLocaleJSONObject.get("localeId"));
```

Comment:
Could we use `LocaleUtil.US` here? :thinking:

Reply:
Done!